cmd/emulator/start/start.go

```diff
@@ -60,6 +60,7 @@ type Config struct {
 	TransactionFeesEnabled bool          `default:"false" flag:"transaction-fees" info:"enable transaction fees"`
 	TransactionMaxGasLimit int           `default:"9999" flag:"transaction-max-gas-limit" info:"maximum gas limit for transactions"`
 	ScriptGasLimit         int           `default:"100000" flag:"script-gas-limit" info:"gas limit for scripts"`
+	BasicContractsOnly     bool          `default:"false" flag:"basic-contracts-only" info:"deploy only basic Flow contracts when emulator starts"`
```

Comment:
This can be up for discussion, but I feel the default should be true.

My reasoning is that this will require no change from anyone that is already using the emulator and is depending on the emulator being in a certain state when it starts.

Reply:
@srinjoyc what do you think? Should we make it an "--include-nft-contracts" flag?

Reply:
I second @janezpodhostnik comment. Although I believe it could be deployed by default we need to be mindful of other devs depending on the current emulator state.